src/libs/actions/Report.js

```diff
@@ -380,6 +380,16 @@ function openReport(reportID, participantList = [], newReportObject = {}, parent
         params.shouldRetry = false;
     }
 
+    // If we open an exist report, but it is not present in Onyx yet, we should change the method to set for this report
+    // and we need data to be available when we navigate to the chat page
+    if (_.isEmpty(ReportUtils.getReport(reportID))) {
+        optimisticReportData.onyxMethod = Onyx.METHOD.SET;
+        optimisticReportData.value = {
+            ...optimisticReportData.value,
+            reportID: reportID.toString(),
+        };
+    }
```

Comment:
This seems to be causing this regression: https://github.com/Expensify/App/issues/20394